orderbook/src/orderbook.rs

```diff
@@ -90,6 +94,10 @@ impl Orderbook {
         Ok(orders)
     }
 
+    pub async fn get_trades(&self, filter: &TradeFilter) -> Result<Vec<Trade>> {
```

Comment:
Should this method be exposed on the orderbook? I feel like trades are conceptually different and would prefer a not to make this component more responsible than it needs to be? As of right now we could directly pass the db (probably some form of interface would make sense to limit the scope we pass around).

Reply:
Sounds about right. Good eye. I will have to look a little closer at this to see what to do here.

Reply:
All done! Weird that I wasn't getting `warning(unused_code)` here before moving it out as a stand alone function...